babeld/babeld.c

```diff
@@ -411,6 +411,7 @@ babel_main_loop(struct thread *thread)
     }
 
     assert(0); /* this line should never be reach */
+    return 0;
```

Comment:
Are we using a different compiler or different version of C?
This isn't required for C99.